server/odc-migrate/src/main/resources/migrate/common/V_4_2_3_1__alter_list_flow_instance_view.sql

```diff
@@ -0,0 +1,18 @@
+--
+-- Alter view `list_flow_instance_view` to support new task types
+--
+create or replace view list_flow_instance_view as
```

Comment:
What is this for? I don’t see any reference to it.

Reply:
It is used in listing flow instance. See the original DDL in V_4_2_0_41__add_list_flow_instance_view.sql and the releated entity is `FlowInstanceViewEntity`